services/s3/src/test/java/software/amazon/awssdk/services/s3/handlers/PutObjectHeaderTest.java

```diff
@@ -77,25 +78,31 @@ public void putObjectBytes_headerShouldContainContentType() {
     @Test
     public void putObjectFile_headerShouldContainContentType() throws IOException {
         File file = new RandomTempFile("test.html", 10);
```

Comment:
Random file calls "deleteOnExit()" internally. I am okay with explicitly deleting the file. I am curious if fortify mentioned this change?

Reply:
No, it didn't mention it. I caught it and didn't know it got deleted automatically.  I will remove it in this case.